subprojects/gst-plugins-bad/sys/msdk/gstmsdkdec.c

```diff
@@ -1019,14 +1019,7 @@ gst_msdkdec_context_prepare (GstMsdkDec * thiz)
     return TRUE;
   }
 
-  /* TODO: Currently d3d allocator is not implemented.
-   * So decoder uses system memory by default on Windows.
-   */
-#ifndef _WIN32
   thiz->use_video_memory = TRUE;
```

Comment:
As we set thiz->use_video_memory as TRUE in gst_msdkdec_init(), can we avoid doing this again here? Also, do you think it is necessary to add a comment when we set this flag as TRUE, just to declare we always use video memory for the decoding process.

Reply:
fixed by add a new commit that delete use_video_memory, thx.